website/content/en/highlights/2021-11-16-CSV-enrichment.md

```diff
@@ -0,0 +1,68 @@
+2021-11-16-CSV-enrichment.md
+---
+date: "2021-11-15"
+title: "CSV enrichment of data"
+description: "A guide to using the new CSV enrichment feature"
+authors: ["barieom", "lucperkins"]
+pr_numbers: []
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+---
+
+# Enrich your data with CSV 
+
+We're excited to share that we've released a new feature that allows users to enrich events flowing through the topology using a CSV file. 
+
+[Enrichment tables][] enables events to have more context and be more readable. This feature works by looking at a single row in a pointed CSV file, allowing users to map the data into the event using the full power of VRL. 
```

Comment:
```suggestion
[Enrichment tables][] is a new concept in Vector that allows you to enrich events from external data sources. To start, we've added the ability to enrich events from a CSV file by looking up a row, or rows, matching provided conditions.  This feature is integrated into the `remap` transform allowing users to map the data into the event using the full power of VRL. 
```

You don't have to take these changes as-is, but I thought it was useful to note that enrichment tables will likely be expanded in the future. We are just starting with CSV enrichment.